src/lib/createHelpers.ts

```diff
@@ -21,9 +31,9 @@ The highlight helper expects a JSON object of the format:
 { "attribute": "name", "highlightedTagName": "mark" }`);
       }
     },
-    snippet(options, render) {
+    snippet(options: string, render: HoganRenderer) {
       try {
-        const snippetOptions = JSON.parse(options);
+        const snippetOptions: Omit<SnippetOptions, 'hit'> = JSON.parse(options);
```

Comment:
Why do we do this? To copy the value or something else?

Reply:
In the `snippet` (and `highlight`) Hogan helper, you don't pass the `hit` because you don't have access to it and we can infer it. Therefore the options that you pass are the snippet options, without the hit.

Reply:
My question was about the JSON.parse